### PR TITLE
Method for creating context for logging

### PR DESCRIPTION
**Pull request by Alexsisukin**
Added a method to create a logging context.
This is necessary in order to be able to name not only elapsed, but also data such as the number of lines in the insert update.
Added functionality since I did not find any clues for the ability to transfer additional parameters to the logger

### CODE REVIEW

File: src/Driver/Driver.php [hunk 1]
```diff
@@ -693,4 +691,24 @@ protected function getDSN(): string
...
+     * @param PDOStatement|null $statement  Statement
+     *
+     * @return array
+     */
+    protected function createLoggerContext(float $queryStart, ?PDOStatement $statement): array
+    {
+        $context = [
+            'elapsed' => microtime(true) - $queryStart,
+        ];
+        if ($statement) {
```
Comment (wolfy-j) [inline src/Driver/Driver.php]: !== null

File: src/Driver/Driver.php [hunk 2]
```diff
@@ -693,4 +691,24 @@ protected function getDSN(): string
...
+
+    /**
+     * Creating a context for logging
+     *
+     * @param float             $queryStart Query start time
+     * @param PDOStatement|null $statement  Statement
+     *
+     * @return array
+     */
+    protected function createLoggerContext(float $queryStart, ?PDOStatement $statement): array
```
Comment (wolfy-j) [inline src/Driver/Driver.php]: maybe `getLoggerContext`? or `defineLoggerContext`? I think this name is fine, but `create` confuses a bit

**Review by wolfy-j — CHANGES_REQUESTED** [review]

File: src/Driver/Driver.php (hunk 1 above)
Comment (Alexsisukin) [reply src/Driver/Driver.php]: @wolfy-j fix

File: src/Driver/Driver.php (hunk 2 above)
Comment (Alexsisukin) [reply src/Driver/Driver.php]: @wolfy-j Thanks for paying attention, corrected to defineLoggerContext

**Pull request merged**